packages/engine-core/src/NodeEngine.ts

```diff
@@ -41,11 +41,23 @@ export interface EngineConfig {
 const knownPlatforms = [
   'native',
   'darwin',
+  'windows',
   'linux-glibc-libssl1.0.1',
   'linux-glibc-libssl1.0.2',
   'linux-glibc-libssl1.1.0',
   'linux-musl-libssl1.1.0',
 ]
+
+const getQueryEngine = (platform: string, prefix: string = __dirname): string => {
```

Comment:
Please rename it to `getQueryEnginePath` and make it a `private` method on the `NodeEngine` class as `getQueryEngine` may suggest that you get an instance of the query engine and having it as a private method on the class is cleaner.

Reply:
Fixed in next commit